internal/clients/nightfall/nightfall_internal_test.go

```diff
@@ -0,0 +1,303 @@
+package nightfall
```

Comment:
These tests are not new they have just moved files `nightfall_test.go => nightfall_internal_test.go`.

New tests have been added in `nightfall_test.go`